### ISSUE TITLE
LuaTuple Type Alias Bug

### ISSUE THREAD

**Issue by grilme99** [roblox-ts/roblox-ts#1504]
```typescript
type R = LuaTuple<[unknown, Script | LocalScript, Actor]>

declare function foo(): R

const [,, actor] = foo()
```

The emit of this looks like the following:
```lua
local _0 = foo()
local actor = _0[3]
```

As you can see, the compiler is trying to index the `LuaTuple` like a table, which is incorrect.

Playground link: https://roblox-ts.com/playground/#code/C4TwDgpgBASlC8UAyBXAhgFRWANhAPANooB2A1iQPYDuJANFAMoDGATgJZjBQA+ylzNDhYcuDAILNglVgF0AfAChFAEwjMcaVtABmpKe0okoOypQAUASgBcsZcyMBnboToM0UmbIQmzVoA

**Comment by osyrisrblx** [comment]
I think we can improve this via symbol checking for `_nominal_LuaTuple`

**Comment by osyrisrblx** [comment]
Whoops, this is already fixed by ea33d384a4d83d88eb05520890cf250aa6d3fa5d
